pkg/daemon/writer.go

```diff
@@ -78,8 +79,10 @@ func (nw *NodeWriter) SetUpdateWorking(client corev1.NodeInterface, node string)
 	return <-respChan
 }
 
-// SetUpdateDegraded Sets the state to UpdateDegraded.
-func (nw *NodeWriter) SetUpdateDegraded(client corev1.NodeInterface, node string) error {
+// SetUpdateDegraded logs the error and sets the state to UpdateDegraded.
+// Returns an error if it couldn't set the annotation.
+func (nw *NodeWriter) SetUpdateDegraded(err error, client corev1.NodeInterface, node string) error {
+	glog.Error("Marking degraded due to: %v", err)
```

Comment:
Think this should be `glog.Errorf` because I'm seeing `Marking degraded due to: %vdaemon can't reconcile this config` :)

Reply:
Thanks, fixed!